09-intermediate-rails/view-helpers.md

```diff
@@ -178,13 +178,13 @@ We've learned about a whole bunch of places to put code: the Controller, the Mod
 | Complex database queries                        | Model (self method)     | Retrieve the first 10 albums, in descending order by number of votes |
 | Interesting action on a database object         | Model (instance method) | Total revenue from all sales for this Product, age of this Album |
 | Checking that a request is reasonable           | Controller              | Filters, strong params |
-| Rendering a small, simple piece of a webpage    | View Helper             | Formatting a date, prefixing the page title with the site title, building a link to a named route |
-| Rendering a big, complicated piece of a webpage | Partial View            | Table of media items, form to create or edit a post |
+| Reusable view specific logic that is ruby-involved; not very HTML involved | View Helper             | Formatting a date, prefixing the page title with the site title, building a link to a named route |
```

Comment:
You say "ruby-involved" but say "HTML involved".  Making them match would be good.

```suggestion
| Reusable view specific logic that is Ruby involved; not very HTML involved | View Helper             | Formatting a date, prefixing the page title with the site title, building a link to a named route |
```
 or

```suggestion
| Reusable view specific logic that is ruby-involved; not very HTML-involved | View Helper             | Formatting a date, prefixing the page title with the site title, building a link to a named route |
```